packages/deno/src/sdk.ts

```diff
@@ -26,6 +26,7 @@ export const defaultIntegrations = [
   new ContextLines(),
   new NormalizePaths(),
   new GlobalHandlers(),
+  new DenoCron(),
```

Comment:
We'll have to make this opt-in because crons will be billed for soon.

I think required importing and using `DenoCron` integration explicitly is good first step.